src/core/lib/iomgr/resolve_address.h

```diff
@@ -43,6 +44,7 @@
 typedef struct {
   char addr[GRPC_MAX_SOCKADDR_SIZE];
   size_t len;
+  bool is_balancer;
```

Comment:
I'd prefer us moving this up one level.

iomgr/resolve_address.h is a low level wrapper around whatever name resolution facility the operating system has.

We're now introducing a new type, which has some metadata about the address stored also. I'm not sure that information belongs in iomgr at all, and would prefer to see it carried in some other data structure, and from some other module.

Proposal: in client_config/, add a grpc_address type:
```
struct grpc_address {
  bool is_balancer;
  grpc_resolved_address sockaddr;  
};
```